packages/shared/src/react/hooks/useOrganization.tsx

```diff
@@ -22,19 +22,24 @@ import {
 import type { PaginatedHookConfig, PaginatedResources, PaginatedResourcesWithDefault } from '../types';
 import { usePagesOrInfinite, useWithSafeValues } from './usePagesOrInfinite';
 
-type UseOrganizationParams = {
+/**
+ * @interface
+ */
+export type UseOrganizationParams = {
```

Comment:
Inside `useOrganization` and `useOrganizationList` you'll notice that I added `export` to the params and return types. I need those to be accessible to typedoc, which requires you to export them.

Since we don't re-export them from `src/react/hooks/index.ts` file they won't surface to the users.